docs/src/main/asciidoc/podman.adoc

```diff
@@ -8,73 +8,37 @@ https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
 :topics: podman,devops,tooling
 include::_attributes.adoc[]
 
-https://podman.io/[Podman] is a daemonless and rootless container engine for developing, managing, and running OCI Containers on your Linux system or other OS.
-If you're using Podman with Quarkus, some one-off setup is needed, but once it's done, you can take advantage of all the Quarkus features.
+https://podman.io/[Podman] is a daemonless and rootless container engine for developing, managing, and running OCI Containers on Linux, Windows and OSX
 
-== Installing Podman
+Podman's approach varies dependent on the operating system you are using and also varied over time dpeendent on version of Podman. Thus albeit Podman comes with a cli based installer and available in various custom package installers (like brew or Linux package mangers) our default recommendation is to use the official Podman Desktop installer. This will ensure you get the best Developer Joy experience and support for your operating system.
```

Comment:
```suggestion
Podman's install approach varies depending on the operating system you are using, and the required steps also change over time depending on the version of Podman. Although Podman comes with a cli-only installer and is available in various package installers (like brew or Linux package managers) our default recommendation is to use the official Podman Desktop installer. This will ensure you get the best Developer Joy experience and support for your operating system.
```

Reply:
"CLI-based installer" implies the installer is a CLI. The CLI installer is actually GUI which installs a CLI, which I tried to capture with "cli-only installer", but there may be a clearer phrasing.